src/styles/header.scss

```diff
@@ -0,0 +1,113 @@
+header {
+    position: fixed;
+    z-index: 999;
```

Comment:
Encapsulating z-index values in a config file or establishing a standardized approach across the application can indeed prevent stacking issues and enhance maintainability.
```
export const zIndex = {
  header: 999,
  modal: 1000,
  tooltip: 1001,
  // Add more as needed for other UI elements
};
```